rero/rero-ils
# admin UI: fix and translations

* NEW Adds a label for search views.
* BETTER Improves the result count message.
* BETTER Improves the patrons roles display.
* FIX Fixes the layout for search views without facets.
* FIX Fixes missing translations.
* BETTER Improves patron types, item types, circulation policies fixtures.
* FIX Pins isort to >=4.3.5,<4.3.10
* Corrects some typos.
* Removes a remain JSON print on the item detailed view template.

Signed-off-by: Igor Milhit <igor.milhit@rero.ch>

----

## To test

- `./script/bootstrap`
- `./script/setup`
- Check the translation in the professional UI.
- Check the layout in the professional UI, with or without facets.
- Check the display of patron roles in the professional UI patrons search (brief views).
- Check the patron types, item types and circulation policies fixtures (name and description).
